server/notification-providers/nostr.js

```diff
@@ -0,0 +1,101 @@
+const NotificationProvider = require("./notification-provider");
+// polyfill for node <= 18
+global.crypto = require("crypto");
+const {
+    relayInit,
+    getPublicKey,
+    getEventHash,
+    getSignature,
+    nip04,
+    nip19
+} = require("nostr-tools");
+// polyfill for node <= 18
+global.WebSocket = require("ws");
```

Comment:
It looks like `ws`'s implementation is completely different to the browser's WebSocket, wondering how it could be possible. 

And the comment said that it is for node <= 18 only, but it looks like it will override regardless of version.

Reply:
Thanks for the review. I added conditional polyfills by node version. Node 14 doesn't have crypto.getRandomValues and I could find a polyfill or other solution for it.